core/src/main/java/Scenes/Lobby.java

```diff
@@ -16,78 +18,349 @@
+          pressedDifficulty = hardButton;
+        } else if (pressedDifficulty != null) {
+          pressedDifficulty.setChecked(false);
+          pressedDifficulty = hardButton;
+        }
+        return true;
+      }
+    });
+    shortButton.addListener(new InputListener() {
+      @Override
+      public boolean touchDown(InputEvent event, float x, float y, int pointer, int button) {
+        if (pressedLength == null) {
+          pressedLength = shortButton;
+        } else if (pressedLength != null) {
+          pressedLength.setChecked(false);
+          pressedLength = shortButton;
+        }
+        return true;
+      }
+    });
+    mediumLengthButton.addListener(new InputListener() {
+      @Override
+      public boolean touchDown(InputEvent event, float x, float y, int pointer, int button) {
+        if (pressedLength == null) {
+          pressedLength = mediumLengthButton;
+        } else if (pressedLength != null) {
+          pressedLength.setChecked(false);
+          pressedLength = mediumLengthButton;
+        }
+        return true;
+      }
+    });
+    longButton.addListener(new InputListener() {
+      @Override
+      public boolean touchDown(InputEvent event, float x, float y, int pointer, int button) {
+        if (pressedLength == null) {
+          pressedLength = longButton;
+        } else if (pressedLength != null) {
+          pressedLength.setChecked(false);
+          pressedLength = longButton;
+        }
+        return true;
+      }
+    });
+
+
   }
 
   /**
    * Adds all of the buttons to the stage so that they are drawn to the screen.
    */
   @Override
   protected void addAllActors() {
-    stage.addActor(playButton);
     stage.addActor(backButton);
+    stage.addActor(playButton);
+    stage.addActor(generalButton);
+    stage.addActor(booksButton);
+    stage.addActor(filmButton);
+    stage.addActor(musicButton);
+    stage.addActor(televisionButton);
+    stage.addActor(videoGamesButton);
+    stage.addActor(sportsButton);
+    stage.addActor(easyButton);
+    stage.addActor(mediumDifficultyButton);
+    stage.addActor(hardButton);
+    stage.addActor(shortButton);
+    stage.addActor(mediumLengthButton);
+    stage.addActor(longButton);
+  }
+
+  /**
+   * Sets the style for when the button is not pressed and when it is using the atlas to find which
+   * textures to use corresponding to the strings 'up' and 'down'. The style font is the font used
+   * for text within buttons and can be changed to whatever is preffered.
+   *
+   * @param up   string corresponding to the atlas texture to use for when the button is not
+   *             pressed.
+   * @param down string corresponding to the atlas texture to use for when the button is pressed.
+   * @return a style with the corresponding 'up' and 'down' textures used for the button and the
+   * font that is set within the method.
+   */
+  private TextButtonStyle setStyle(String up, String down) {
```

Comment:
"setStyle" isn't an appropriate name for this method, as it implies that it's used in setting the style for this Lobby class.  "createStyle" would be more appropriate, since you're creating and returning a new style object.